packages/in_app_purchase/in_app_purchase_storekit/darwin/in_app_purchase_storekit/Sources/in_app_purchase_storekit_objc/include/in_app_purchase_storekit_objc/FLTMethodChannelProtocol.h

```diff
@@ -2,11 +2,14 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+#include <TargetConditionals.h>
```

Comment:
Is this needed? If I remove this line, I can still build fine locally using both SwiftPM and CocoaPods.

Reply:
See: https://github.com/flutter/packages/pull/8469/files#r1929267019

Reply:
This is needed since `TARGET_OS_OSX` is used right below.

Reply:
Closing as this one is indeed needed